requirements/edx/base.txt

```diff
@@ -199,5 +199,9 @@ edx-django-release-util==0.2.0
 # modulestore inspection
 py2neo==3.1.2
 
+# Used for printing the Edraak certificates
+python-bidi==0.3.4
```

Comment:
You can use my [python-arabic-reshaper](https://github.com/ahmedaljazzar/python-arabic-reshaper) if you want.